src/cpu/core_dyn_x86/cache.h

```diff
@@ -137,8 +137,10 @@ class CodePageHandler : public PageHandler {
 		addr&=4095;
 		if (host_readw(hostmem+addr)==(Bit16u)val) return;
 		host_writew(hostmem+addr,val);
-		if (!*(Bit16u*)&write_map[addr]) {
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).